Extras/Scripts/sign-bundled.sh

```diff
@@ -158,18 +158,12 @@ APPID=`/usr/libexec/PlistBuddy -c "Print :CFBundleIdentifier" "${BUILT_INFO_PLIS
 echo "Resigning bundled items."
 echo "App bundle is $APPID."
 
-if [[ "$CODE_SIGN_IDENTITY" == "" ]] ; then
+if [[ "$EXPANDED_CODE_SIGN_IDENTITY" == "" ]] ; then
   echo "App not signed, using default identity."
-  CODE_SIGN_IDENTITY="3rd Party Mac Developer Application"
+  EXPANDED_CODE_SIGN_IDENTITY="3rd Party Mac Developer Application"
 fi
 
-# Bit of a hack: according to the codesign tool, Mac Developer is ambiguous (also matches 3rd Party Mac Developer)
-# Mac Developer: shouldn't be, so we change it if necessary.
-if [[ "$CODE_SIGN_IDENTITY" == "Mac Developer" ]] ; then
-  CODE_SIGN_IDENTITY="Mac Developer:"
-fi
```

Comment:
As a result of the above, this hack is no longer needed. Xcode will decide which cert for us.